testcrate/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "testcrate"
+version = "0.1.0"
+authors = ["Wim Looman <wim@nemo157.com>"]
+
+[dependencies]
+uom = { path = ".." }
+
+[[example]]
```

Comment:
I'd like to see distinct examples between the two crates. Currently if there is an example that fails to compile it will show up for both `uom` and `testcrate`.

Reply:
Unfortunately I don't see a way to do that nicely.

My first thought was to remove the examples from `uom` itself and only build them as part of `testcrate`, but that would make it impossible to run `cargo run --example mks` from the root.

Not compiling the example as a part of `testcrate` would defeat the purpose of `testcrate`, to make sure that the examples are not depending on the un-intentional propagation of features from library to example.